CONTRIBUTING.md

```diff
@@ -102,6 +102,10 @@ To auto fix eslint rule failures that are autofixable run `npm run test:lint --
 
 * Write tests for your changes. A feature is not considered done until it has tests and/or a test plan. It does not matter if code coverage shows 100%, tests are expected for *all* changes. This includes [integration tests](./__tests__/integration/README.md) as needed.
 
+### Getting in contact
```

Comment:
```suggestion
### Getting In Contact
```
To keep consistent with the rest of the document, could we capitalize this please?